gtnh-modpack.json

```diff
@@ -488,7 +488,7 @@
       "filename": "EnderZoo-1.7.10-1.0.23.jar",
       "download_url": "https://api.github.com/repos/GTNewHorizons/EnderZoo/releases/assets/56698392",
       "browser_download_url": "https://github.com/GTNewHorizons/EnderZoo/releases/download/1.0.23/EnderZoo-1.7.10-1.0.23.jar",
-      "license": "Creative Commons Zero v1.0 Universal",
+      "license": "CC0 1.0 Universal Public Domain Dedication.",
```

Comment:
Why do you use full name for other CC licenses but not this one?

Reply:
I skip the brackets on all the licenses because it's just an abbreviation (the other licenses on their pages are suffixed by, for example, (CC BY-SA 3.0), this is just restating the license). The PD bit is included because CC0 is just using the Creative Commons license text, it should still be noted that the license is PD to the outward observer.